tutor-multi-chart/values.yaml

```diff
@@ -2,24 +2,23 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+ingress-nginx:
+  # Use ingress-nginx as a default controller.
+  enabled: true
 
-# In the future, configuration for tutor-multi-chart will go here:
-
-# replicaCount: 1
-
-# image:
-#   repository: nginx
-#   pullPolicy: IfNotPresent
-#   # Overrides the image tag whose default is the chart appVersion.
-#   tag: ""
-
-# imagePullSecrets: []
-# nameOverride: ""
-# fullnameOverride: ""
-
+cert-manager:
+  # Use cert-manager as a default certificate controller.
+  enabled: true
```

Comment:
The cert-manager docs say:

> Be sure never to embed cert-manager as a sub-chart of other Helm charts; cert-manager manages non-namespaced resources in your cluster and care must be taken to ensure that it is installed exactly once.

But they also [provide clear instructions on _how_ to install it as a sub-chart](https://cert-manager.io/docs/installation/helm/#installing-cert-manager-as-subchart) so I guess it's fine?

I think we should probably just put something in the README which says "If you already have cert-manager on your cluster, be sure to set `cert-manager.enabled: false` for this chart, to avoid any conflicts.

What do you think?

Reply:
At first glance seems like the main concern is losing track of whether you have cert-manager or not and installing it twice by mistake. I added a section in the README that mentions this.

This actually got me thinking about having a large amount of different resources living under the same namespace and if it's that what we want (autoscaling, monitoring, etc). The cert-manager docs mention that we could install each dependency in its own namespace if each chart allows it.

Reply:
> This actually got me thinking about having a large amount of different resources living under the same namespace and if it's that what we want (autoscaling, monitoring, etc). The cert-manager docs mention that we could install each dependency in its own namespace if each chart allows it.

I'm open to either approach. For now I think one namespace is easier because of how helm works but it may not be the best.